README.md

```diff
@@ -189,8 +189,7 @@ make dfu
 
 This will flash both the images to the device using the dfu-util program.
 
-If you get a permission error, make sure you followed the [#Join the plugdev
-group] instructions above for this user. You shouldn't need to run this command
+If you get a permission error, make sure you followed the [plugdev group](#join-the-plugdev-group) instructions the [plugdev group](#join-the-plugdev-group) instructions above for this user. You shouldn't need to run this command
```

Comment:
Good catch, thanks for fixing this for me @grgustaf ! The [plugdev group' link appears twice though now. Can you correct this?